package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "Simple leaderboard app that shows a list of the recent scores and updates through a refresh button. Built with Javascript, HTML, CSS and Webpack.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
Great work including the build command for your application, but it lacked a way to start your application. Your script may look like the following `"start": "webpack --config webpack.config.js --watch"`